proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,531 @@
+available for contribution and what stage they are in. Figure A demonstrates how
+a request and contribution will proceed through the improved process.
+
+We will also publish an improved contributing guide to document the review
+process, acceptance criteria, and expectations for communication. We will use
+existing issue labeling in conjunction with a GitHub Project to provide
+transparent status to the community to indicate what is available for
+contribution and to track the progress of a contribution.
+
+The following sections detail the proposed improvements to the existing process.
+
+### Intake
+
+We will modify the the intake process to set contribution expectations for the
+community. A contribution is initiated by opening a GitHub issue. The GitHub
+issue captures information necessary to make the contribution, such as:
+
+1. A description of the requested change.
+1. The number of users in the community that request the change, measured
+   through GitHub :+1: reactions.
+1. Design discussions, including use cases, edge cases, and alternative
+   solutions.
+
+Contributions made without a GitHub issue will be subject to closure by the
+maintainers.
+
+Maintainers will regularly review open GitHub issues to determine if they are
+ready for community contribution. The decision to label an issue as ready for
+community contribution is ultimately up to the maintainers. Depending on the
+issue type (bug or feature request) and area (code or documentation), the
+maintainers will use a set of criteria to decide if the issue is ready for
+contribution. This may include:
+
+1. The change is feasible for a community member to contribute. See [the
+   appendix](#changes-not-available-for-contribution) for details.
+1. Feature requests must have at least 10 :+1: reactions.
+1. The maintainers agree with the implementation plan.
+
+If the criteria are not satisfied, the maintainers will comment on the GitHub
+issue to request more information and to update the current status or next
+steps. Any design discussions (e.g., public interface design, edge cases,
+backwards compatibility concerns) must be resolved prior to making a
+contribution so that feedback during the review process focuses on the
+implementation of the agreed upon design.
+
+Issues that do not receive enough :+1: reactions and do not have any activty for
+a year are considered stale. The maintainers will make a request for comment
+from the community. Without further interaction, the issue will be closed.
+
+### Contribution-ready
+
+If the criteria for ready for contribution are satisfied, the issue will be
+labelled to identify it as ready to contribute. This new stage clearly
+identifies which issues the maintainers have determined are ready for
+contribution and makes them visible for users can discover them. Any issue that
+has been marked as available is open for a community contribution through a
+GitHub pull request.
```

Comment:
Just some suggestions on how to rephrase this to tighten the wording a bit (especially what are the commitments made when it enters contribution ready) and the ordering of content while still warranting the content as its own section:

A GitHub issue enters the "Contribution-ready" stage when a maintainer publicly labels the issue as ready for contribution. This stage indicates:

* Any user in the community is welcome to open a contribution via a GitHub pull request in order to address the GitHub issue.
* If a pull request is opened and meets all review prerequisites (maybe add a link to a later section in the proposal here?), maintainers will review the contribution and work with the contributor to get the contribution merged at some point in time (maybe add a link to the review queue section here?).

All issues that are at the "Contribution-ready" stage are listed publicly to allow the community to discover which issues they can submit contributions for.

Reply:
I made some changes, but don't agree with adding links to other sections. It should read linearly in this case, and not jump around.